config/locales/activemodel.en.yml

```
@@ -11,3 +11,5 @@ en:
             csv_file:
               invalid_extension: "is invalid, must have .csv extension."
               invalid_count: "is invalid, must have 1-1000 keywords."
+        search_result:
+          not_found: "No results found."
```

```suggestion
          not_found: "Search result not found."
```

---

A complete sentence would be better:

```suggestion
          not_found: "No search results found."
```

or 


```suggestion
          not_found: "There were no search results found."
```